src/OpenTK.GraphicsLibraryFramework/Interfaces/IGLFW.cs

```diff
@@ -24,128 +24,185 @@ internal interface IGLFW
         // Still missing in documentation
 
         /// <summary>
+        /// <para>
         /// This function initializes the GLFW library. Before most GLFW functions can be used,
         /// GLFW must be initialized, and before an application terminates GLFW should be terminated in order to
         /// free any resources allocated during or after initialization.
+        /// </para>
+        /// <para>
         /// If this function fails, it calls <see cref="Terminate"/> before returning.
+        /// </para>
+        /// <para>
         /// If it succeeds, you should call <see cref="Terminate"/> before the application exits.
+        /// </para>
+        /// <para>
         /// Additional calls to this function after successful initialization
         /// but before termination will return <c>true</c> immediately.
-        /// </summary>1
+        /// </para>
+        /// </summary>
         /// <returns><c>true</c> if successful, or <c>false</c> if an error occurred.</returns>
         /// <remarks>
+        /// <para>
         /// OS X: This function will change the current directory of the application
         /// to the Contents/Resources subdirectory of the application's bundle, if present.
-        ///
+        /// </para>
+        /// <para>
         /// This function must only be called from the main thread.
-        ///
-        /// TODO: Possible errors include <see cref="ErrorCode.PlatformError"/>.
+        /// </para>
+        /// <para>
+        /// Possible errors include <see cref="ErrorCode.PlatformError"/>.
+        /// </para>
         /// </remarks>
-        /// <seealso cref="Terminate"/>
```

Comment:
Why did you remove that one? It is also found in the original GLFW documentation, and does kinda make sense.
Edit: for all the other seealso that were removed -> the same question

Reply:
I personally don't think it's necessary in most cases, since most of the time, the thing seealso mentions is already mentioned in the main summary text. Also, (at least in Rider) the seealso text is just dumped at the bottom, with no link or anything. So in that case, that seealso just appends "Terminate()" to the end with no context.